### DISCUSSION TITLE
Model output frequency - daily or hourly?

### DISCUSSION THREAD

**Discussion by klo-h2o** [NASAWaterInsight/NLDAS-3#3] [Q&A] [answered]
NLDAS data users and stakeholders: Do you want daily or hourly model output? How will you use the data? Why hourly or daily?

According to my sources, either way, we will provide model forcing (precipitation, temperature, winds, radiation, humidity) at an hourly resolution, as we need that temporal resolution to RUN the land-surface model.

**Comment by tm-parris** [comment]
We are considering the use of NLDAS3 as part of an effort to develop/assess drought indicators that to not rely on assumptions of stationarity over a baseline period.  At this point, we would only use daily data/summaries.  At some point, we would also like to look at changes in diurnal patterns over time.  If we get to that, the hourly data would be useful as well. 

[Though, I think this question is moot.  It looks like you are posting both daily and hourly data.  This is just my way of saying, "hello."]

**Comment by aputman-usgs** [comment]
I know this is an old discussion, but wanted to +1 the hourly choice you guys made. We need hourly for snow modeling. I think hourly is also best for hydrologic modeling.

**Comment by dmocko** [comment, marked as answer]
We plan on continuing to provide the surface meteorological forcing data (8 variables: T/q/P/SW/LW/U/V/Precip) as hourly,
as well as with daily and monthly averages.

The land-surface model (LSM) output data will only be daily-averages (variables such as soil moisture, snow, evaporation, etc.).

For the full list of output variables, please see our NLDAS-3 page:  https://ldas.gsfc.nasa.gov/nldas/v3

**Reply by jessiezhang7** [reply]
I just came across this discussion. We have been using hourly soil moisture and soil temperature from NLDAS to initialize WRF-Chem. When will you stop generating NLDAS-2 data? Is it possible to provide hourly data for the LSM output from NLDAS-3?

**Reply by dmocko** [reply]
Hi @jessiezhang7 

NLDAS-2 is a collaboration project, led by NOAA and NASA.  It runs in operations at NOAA every day.  It's up to NOAA when NLDAS-2 will no longer be run.

NLDAS-3 is a NASA-led project.  For the retrospective period (2001 to ~2026), we will only make daily-average LSM output available.  Once this phase is running routinely, we are considering distributing only the near real-time data as hourly instead of daily.

Please e-mail me at my NASA address so we can continue this conversation and talk about your workflow requirements.

David
